### PR TITLE
Use uniform initializer for recurrent weights

### PR DESCRIPTION
**Pull request by EdeMeijer**
Addresses https://github.com/batzner/indrnn/issues/1

### CODE REVIEW

File: ind_rnn_cell.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from tensorflow.python.layers import base as base_layer
 
 
-class IndRNNCell(rnn_cell_impl._LayerRNNCell):
+class IndRNNCell(rnn_cell_impl.RNNCell):
```
Comment (batzner) [inline ind_rnn_cell.py]: I am new to custom RNN development. Can you explain, why you changed it to `RNNCell`?

File: ind_rnn_cell.py [hunk 2]
```diff
@@ -60,13 +60,16 @@ def build(self, inputs_shape):
       "input_%s" % rnn_cell_impl._WEIGHTS_VARIABLE_NAME,
       shape=[input_depth, self._num_units])
 
+    recurrent_init = init_ops.random_uniform_initializer(
```
Comment (batzner) [inline ind_rnn_cell.py]: Would it make sense to make the `recurrent_init` a constructor argument like in `GRUCell` and `LSTMCell` with the default being uniform init? It seems to make a big difference, so people might want to experiment with it.

File: ind_rnn_cell.py (hunk 1 above)
Comment (EdeMeijer) [reply ind_rnn_cell.py]: Sorry, I did a quick search and found that _LayerRNNCell is the way to go, I changed it to make it work with my slightly older install of TF. Will revert.

File: ind_rnn_cell.py (hunk 2 above)
Comment (EdeMeijer) [reply ind_rnn_cell.py]: Definitely, will look into that.

**Pull request merged**